deepchem/models/tests/test_cnn.py

```diff
@@ -137,3 +132,33 @@ def test_cnn_regression_uncertainty(self):
     pred, std = model.predict_uncertainty(dataset)
     assert np.mean(np.abs(y - pred)) < 0.3
     assert noise < np.mean(std) < 1.0
+
+
+@pytest.mark.torch
+def test_cnn_torch():
+
+  from deepchem.models.torch_models.cnn import CNN
```

Comment:
Fails PyTorch Test in `CI` says:
`FAILED deepchem/models/tests/test_cnn.py::test_cnn_torch - ModuleNotFoundErro...`